TeeWhyKay/ChompFood
# Fix/hide orders with zero items

## Why

If user initially put items in their cart, and then after that deleted them all, they wont be shown the "Your cart is empty".
They will also see the food item with Quantity: 0 in the cart page.

## What

Now, they will see "Your cart is empty" if they deleted everything.
They also won't see the food item with Quantity: 0 any more if they have other food items instead.
The other food items will display as normal.

## How to Test

Go to restaurant page with food lists, add two different food items, and then delete one of it with quantity: 0.
Go to the cart, you should see the item that is not deleted.

Go back to the restaurant page, and delete the remaining food item, then go to My Cart again.
You should see "Your cart is empty".
